client/js/lounge.js

```diff
@@ -646,7 +674,17 @@ $(function() {
 		}
 	});
 
-	socket.on("names", renderChannelUsers);
+	socket.on("names", function(data) {
+		renderChannelUsers(data);
+
+		// Condensed actions support
+		setTimeout(function() {
```

Comment:
Condensed actions is not in master, nor does using setTimeout in event based code is a good idea.